libby/form/ListBox.lib.js

```diff
@@ -163,4 +163,33 @@ describe('ListBox', () => {
       </ListBox>
     </form>
   ));
+
+  add('as a controlled component', () => {
+    const [value, setValue] = React.useState(2);
+
+    function handleClick() {
+      if (value === 4) {
+        setValue(1);
+      } else {
+        setValue(value + 1);
+      }
+    }
+    return (
+      <>
+        <ListBox
```

Comment:
Should probably `setValue()` on change, values get out of syncwhen using the increment button and then choosing a different option by clicking

Reply:
Hmm this is probably a bug with the component – the ticket is to fix the listbox when only value is controlled

Reply:
Oh nvm i see what youre saying